charts/bridgeai-prediction-service/Chart.yaml

```
@@ -19,10 +19,21 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.1.2
```

```suggestion
version: 0.2.0
```

We are adding a new feature.